King-Benx/news-watchtower-5
# custom launch icon [Delivers #164798127]

https://www.pivotaltracker.com/story/show/164798127

**Overview**
Gives the application a custom launch icon. 

**Acceptance Criteria**
- Application should work as usual.
- Application should have a custom launch icon instead of the default icon.

**How to manually test**
- Clone repo.
- Build the app ``./gradlew build````
- Launch the application.
- Leave the app and check the home screen.

**Story**

[#164798127]
